pyccl/tests/test_baryons_vd19.py

```diff
@@ -8,7 +8,7 @@
 # Set up the cosmological parameters to be used
 cosmo = ccl.Cosmology(Omega_c=0.27, Omega_b=0.045, h=0.67,
                       A_s=2.1e-9, n_s=0.96,
-                      transfer_function='boltzmann_class')
+                      transfer_function='boltzmann_camb')
```

Comment:
while we're at it, can we make this 'eisenstein_hu' so the test is a bit faster?

Reply:
Done.